### PR TITLE
Group all cpu usage per core statistics under cpus

### PR DESCRIPTION
**Pull request by monicasarbu**
The goal of this PR is to group all the cpu usage per core statistics under `cpus` and add a configuration option `cpu_per_core` to enable it. By default it is set to false as it exports a lot of data. 

The number of fields that are exports depends on the number of cores, so for these data we are not providing sample dashboards as it depends on the number of cores.

This is the first change out of multiple changes requested by #424.

### CODE REVIEW

File: topbeat/beat/config.go [hunk 1]
```diff
@@ -4,9 +4,10 @@ type TopConfig struct {
 	Period *int64
 	Procs  *[]string
 	Stats  struct {
-		System     *bool
-		Proc       *bool
-		Filesystem *bool
+		System       *bool
+		Proc         *bool
+		Filesystem   *bool
+		Cpu_per_core *bool
```
Comment (ruflin) [inline topbeat/beat/config.go]: Can you use here `CpuPerCore *bool yaml:"cpu_per_core" to make sure we get the right config? We should also add it for the above 3 configs as we did in the other beats to make it clear, the values are coming from a yaml doc.

In the example above the backticks are missing, as they are also use to mark some code :-( here is an example: https://github.com/elastic/filebeat/blob/master/config/config.go#L39

Comment (monicasarbu) [inline topbeat/beat/config.go]: Yes, I agree with you it's better to add "yaml". I didn't reach the point to make nicer the code.

Comment (ruflin) [inline topbeat/beat/config.go]: @monicasarbu Sorry, was too early. No review label yet ;-)

**Comment by monicasarbu** [comment]
This feature is implemented only for Unix systems, so I am disabling the test for windows in order for the tests to pass on windows.

**Comment by monicasarbu** [comment]
Ready for review.

File: topbeat/docs/fields.asciidoc [hunk 2]
```diff
@@ -169,6 +169,78 @@ type: int
...
 
 
+=== cpus Fields
+
+This group contains cpu usage per core statistics.
+
+
+=== cpui Fields
+
+This group contains cpu usage statistics of the core i, where i is the core number
```
Comment (tsg) [inline topbeat/docs/fields.asciidoc]: I think it would look better to use cpuX with X being the variable rather then "i".

Comment (monicasarbu) [inline topbeat/docs/fields.asciidoc]: done

**Pull request merged**